docs/semgrep-app/getting-started-with-semgrep-app.md

```diff
@@ -99,17 +89,113 @@ Scanning is Semgrep's primary operation. When you first sign into Semgrep App, i
+* Your GitHub or GitLab account that you add to Semgrep App for scanning. Projects from GitHub or GitLab are integrated through Semgrep App.
+* A local git repository in your machine. Projects from your local machine are integrated through Semgrep CLI.
 
+Semgrep App can run scans on many projects with rules set in the Rule Board. First-time Semgrep App users scan projects with pre-selected rules chosen based on the repository's language and framework. To view these pre-selected rules, see the [Registry default ruleset](https://semgrep.dev/p/default).
 
-To add a project:
+Over time, users modify the Rule Board with rules specific to their codebase's security or business goals.  
+
+#### Option A: Adding a local repository through Semgrep CLI
+
+Adding a local repository is considered a standalone scan. This means the scan is manually triggered and is not continuous, unlike a CI job. 
+
+**Prequisite:** Semgrep CLI must be installed. See [Getting started with Semgrep CLI](../../getting-started).
+
+1. Ensure you are signed in to Semgrep App.
+2. Click **Projects** on the left sidebar.
+3. Click **Scan new project** > **Run a scan locally**.
+4. Log in to Semgrep from the CLI:
+``` 
+semgrep login
+```
+6. Follow the instructions on the CLI.
+5. After logging in, run a scan by entering the following command. This command sends the findings to Semgrep App.
+```
+semgrep ci
+```
+6. View your project's [findings](https://semgrep.dev/orgs/-/findings).
+
+:::note
+Scans from local repositories do not access their corresponding remote repositories. For this reason, links to specific lines of code in the Findings page are not created. See [Linking local scans to their remote repositories](#linking-local-scans-to-their-remote-repositories) for a workaround.
+:::
+
+##### Linking local scans to their remote repositories 
+
+![Screenshot of findings page snippet with no hyperlinks](../img/findings-no-hyperlinks.png "Screenshot of findings page snippet with no hyperlinks")
+*Figure 1.* Partial screenshot of findings page with no hyperlinks.
+
+Local repository scans require additional configuration to link to their corresponding remote repositories. Set up environment variables within your command line to configure cross-linking between local and remote repositories within Semgrep App.
+
+To set up environment variables:
+
+1. Ensure that your current working directory is the root of the repository to create links for. 
+2. Set up the `SEMGREP_REPO_URL`:
+    1. Retrieve the URL by navigating to your online repository. Copy the value in the address bar. This is your `URL_ADDRESS`.
+    2. Set the variable by entering the text below, substituting `URL_ADDRESS` with the value from the previous step.

```

Comment:
Where should the user enter this text? I assume it should be entered into the command line and think if that’s the case, the user should be explicitly instructed as such (at least for the first variable, maybe not for all).

Reply:
Yup, we should make sure they know they need to set up shell environment variables

Reply:
I will make it clearer. Maybe I should even have a screenshot...

Reply:
Have added 2 screenshots to make it super clear.